analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpSymbolUsageCollector.cs

```diff
@@ -159,7 +159,9 @@ public override void VisitAttribute(AttributeSyntax node)
 
         public override void VisitIdentifierName(IdentifierNameSyntax node)
         {
-            if (IsKnownIdentifier(node.Identifier))
+            if (IsKnownIdentifier(node.Identifier)
+                || knownSymbolNames.Contains($"get_{node.Identifier.ValueText}")
+                || knownSymbolNames.Contains($"set_{node.Identifier.ValueText}"))
```

Comment:
This is confusing. We add the accessors as a known symbol but then track the usage of the property. I see two different ways of improving this:
1. Either remove this change, but when creating the `CSharpSymbolUsageCollector`, pass the property symbol instead of the accessor—something like `privateSymbols.Select(x => x is IMethodSymbol { AssociatedSymbol: IPropertySymbol property } ? property : x)` in `UnusedPrivateMember` line 87.
2. Track the usage of the accessors in addition to tracking the property usage.